src/content/api/bank-account-approvals.mdx

```diff
@@ -0,0 +1,356 @@
+---
+title: Bank Account Approvals
+description: Bank Account Approval model and related endpoints
+draft: true
+nav:
+  path: API
+  order: 17
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import CodeGroup from '../../components/CodeGroup.astro';
+import Badge from '../../components/Badge.astro';
+
+A Bank Account Approval represents any added authorization on a [Bank Account](/api/bank-accounts).
+This may be an approval from Centrapay or a consent from a Centrapay [Account](/api/accounts) owner
+that allows access to a third-party system. See [Bank Account Approval Types](#bank-account-approval-types).
+
+## Bank Account Approval Model
+
+### Attributes
+
+<Properties>
+  <Property name="id" type="string">
+    The Bank Account Approval's unique identifier.
+  </Property>
+
+  <Property name="bankAccountId" type="string">
+    The id of the associated [Bank Account](/api/bank-accounts).
+  </Property>
+
+  <Property name="accountId" type="string">
+    The id of the owning Centrapay [Account](/api/bank-accounts).
```

Comment:
The link is wrong here

Reply:
The accounts page has already been merged in so I think this one is all good